templates/config.cli.erb

```diff
@@ -83,3 +83,38 @@ catch
 /subsystem=keycloak-server/spi=userCache/provider=default/:remove
 /subsystem=keycloak-server/spi=userCache/provider=default/:add(enabled=<%= scope['keycloak::user_cache']%>)
 end-try
+<%- if scope['keycloak::operating_mode'] == 'clustered' && scope['keycloak::enable_jdbc_ping'] -%>
+if (outcome != success) of /subsystem=jgroups/stack=tcp/protocol=JDBC_PING:read-resource
+<%- if scope['keycloak::datasource_driver'] == 'postgresql' -%>
+/subsystem=jgroups/stack=tcp/protocol=JDBC_PING: add(add-index=0, data-source="KeycloakDS", properties=[initialize_sql="CREATE TABLE IF NOT EXISTS JGROUPSPING ( own_addr varchar(200) NOT NULL, cluster_name varchar(200) NOT NULL, created TIMESTAMP DEFAULT CURRENT_TIMESTAMP, ping_data BYTEA, constraint PK_JGROUPSPING PRIMARY KEY (own_addr, cluster_name))"])
+<%- end -%>
+<%- if scope['keycloak::datasource_driver'] == 'mysql' -%>
+/subsystem=jgroups/stack=tcp/protocol=JDBC_PING: add(add-index=0, data-source="KeycloakDS", properties=[initialize_sql="CREATE TABLE IF NOT EXISTS JGROUPSPING (own_addr varchar(200) NOT NULL, cluster_name varchar(200) NOT NULL, updated TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP, ping_data varbinary(5000) DEFAULT NULL, PRIMARY KEY (own_addr, cluster_name)) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin"])
+<%- end -%>
+end-if
+if (outcome == success) of /subsystem=jgroups/stack=tcp/protocol=MPING:read-resource
+/subsystem=jgroups/stack=tcp/protocol=MPING: remove()
+end-if
+if (outcome == success) of /subsystem=jgroups/stack=tcp/protocol=FD_SOCK:read-resource
```

Comment:
This specific condition can be removed.  The remove + add makes the condition unnecessary.

Reply:
Done it. I `rebase` -i and `fixup` the commit. Let me know if that works for you